setup_virtualenv.sh

```diff
@@ -29,10 +29,10 @@ function setup_virtualenv() {
     VIRTUALENV_PIP="${VIRTUALENV_DIR}/bin/pip"
 
     echo "Installing dependencies into virtual environment..."
-    ${VIRTUALENV_PIP} install --quiet --upgrade ${WHEEL_CACHE_DIR}/compliance_checker-*.whl
-    ${VIRTUALENV_PIP} install --quiet -c constraints.txt --upgrade ${WHEEL_CACHE_DIR}/cc_plugin_imos-*.whl
-    ${VIRTUALENV_PIP} install --quiet -c constraints.txt --upgrade ${WHEEL_CACHE_DIR}/aodncore-*.whl
-    ${VIRTUALENV_PIP} install --quiet -c constraints.txt -r requirements.txt
+    ${VIRTUALENV_PIP} install  --upgrade ${WHEEL_CACHE_DIR}/compliance_checker-*.whl
+    ${VIRTUALENV_PIP} install -c constraints.txt --upgrade ${WHEEL_CACHE_DIR}/cc_plugin_imos-*.whl
+    ${VIRTUALENV_PIP} install -c constraints.txt --upgrade ${WHEEL_CACHE_DIR}/aodncore-*.whl
+    ${VIRTUALENV_PIP} install -c constraints.txt -r requirements.txt
```

Comment:
Did you mean to commit this as well? If so it should be in a separate PR.

Reply:
ok will do

Reply:
Yeah don't trash this file please... it's like getting your car back from the mechanic missing the steering wheel ;)